apps/src/redux/instructions.js

```diff
@@ -116,6 +117,12 @@ export default function reducer(state = instructionsInitialState, action) {
     });
   }
 
+  if (action.type === SET_FEEDBACK) {
+    return _.assign({}, state, {
```

Comment:
Slight preference for Object.assign over _.assign (there were historical reasons why were were using _.assign instead, which no longer apply)